moped-editor/.env-cmdrc

```diff
@@ -4,7 +4,7 @@
     "REACT_APP_AWS_COGNITO_USER_POOL_ID": "us-east-1_U2dzkxfTv",
     "REACT_APP_AWS_COGNITO_APP_CLIENT_ID": "3u9n9373e37v603tbp25gs5fdc",
     "REACT_APP_HASURA_ENDPOINT": "http://localhost:8080/v1/graphql",
-    "REACT_APP_API_ENDPOINT": "http://localhost:5000",
+    "REACT_APP_API_ENDPOINT": "http://localhost:5001",
```

Comment:
on my machine port 5000 was in use by my airplay listener, so i figured it wouldn't hurt to make this change permanent.

Reply:
Yes! Thank you. I've bumped into this several times in the past, and it boggles the mind that apple settled on port 5000 instead of something privileged (<1024) for a listening port for a system service. I wonder to what degree this reveals something about their tech stack.